examples/decoding/plot_decoding_time_generalization_conditions.py

```diff
@@ -25,10 +25,14 @@
 # License: BSD (3-clause)
 
 import numpy as np
+import matplotlib.pyplot as plt
 
 import mne
 from mne.datasets import sample
-from mne.decoding import GeneralizationAcrossTime
```

Comment:
We don't deprecate it, not now at least.

We'll refactor the GeneralizationAcrossTime in a different PR, this is a much bigger job.